### PR TITLE
@types/chrome - Adding playbackRate property to type definition for LoadRequest

### PR DESCRIPTION
**Pull request by pjjanak**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.media.LoadRequest#playbackRate

### CODE REVIEW

File: types/chrome/chrome-cast/index.d.ts [hunk 1]
```diff
@@ -766,6 +766,7 @@ declare namespace chrome.cast.media {
         currentTime: number;
         customData: Object;
         media: chrome.cast.media.MediaInfo;
+	playbackRate: number;
```
Comment (DangerBotOSS) [inline types/chrome/chrome-cast/index.d.ts:769]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/chrome/tsconfig.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     },
     "files": [
         "index.d.ts",
-        "test/index.ts"
+        "test/index.ts",
+	    "test/chrome-cast.ts"
```
Comment (DangerBotOSS) [inline types/chrome/tsconfig.json:23]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/chrome/chrome-cast/index.d.ts [hunk 3]
```diff
@@ -766,13 +720,13 @@ declare namespace chrome.cast.media {
         currentTime: number;
         customData: Object;
         media: chrome.cast.media.MediaInfo;
+        playbackRate: number;
```
Comment (domdomegg) [inline types/chrome/chrome-cast/index.d.ts:723]:
```suggestion
        playbackRate: number | undefined;
```

as per https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.media.LoadRequest#playbackRate

**Review by domdomegg — CHANGES_REQUESTED** [review]
As per code comment

File: types/chrome/chrome-cast/index.d.ts (hunk 3 above)
Comment (nicolas377) [inline types/chrome/chrome-cast/index.d.ts:723]: Both of these need to be marked as optional, as per the docs specifying that they can be undefined.
```suggestion
        media?: chrome.cast.media.MediaInfo | undefined;
        playbackRate?: number | undefined;
```

**Review by nicolas377 — CHANGES_REQUESTED** [review]

File: types/chrome/chrome-cast/index.d.ts (hunk 3 above)
Comment (pjjanak) [reply types/chrome/chrome-cast/index.d.ts:723]: Unless I'm misreading, `media` must be non-null, as per the documentation. `playbackRate` on the other hand is nullable.

Comment (nicolas377) [reply types/chrome/chrome-cast/index.d.ts]: whoops! sorry about that!

**Review by domdomegg — APPROVED** [review]

**Review by nicolas377 — APPROVED** [review]
LGTM!

**Comment by pjjanak** [comment]
Ready to merge

**Pull request merged**